### DISCUSSION TITLE
Documentation on creating builds, releases, docker images.

### DISCUSSION THREAD

**Discussion by srameshr** [plausible/analytics#576] [Anything Else]
Is there any documentation on the following things?
1. Creating new builds or is it the same as the one in the official Phoenix documentation deployment guide (https://hexdocs.pm/phoenix/deployment.html#content)?
2. Creating development and production releases.
3. Creating a new Docker image from the local updated source.

**Comment by ukutaht** [comment]
Hey. Sorry no documentation. I create builds for staging/testing like so:

```
$ docker build -t plausibe/analytics:dev .
$ docker push plausible/analytics:dev
```

I guess you'll need your own registry to push to but this should work.
